src/shopping.py

```diff
@@ -0,0 +1,202 @@
+import os
+from tinydb import TinyDB, Query
+from tabulate import tabulate
+
+current_dir = os.path.dirname(__file__)
+supermarket_db_path = os.path.join(
+    current_dir, "../databases/supermarket.json"
+)
+order_db_path = os.path.join(current_dir, "../databases/order.json")
+
+supermarket = TinyDB(supermarket_db_path)
+order_db = TinyDB(order_db_path)
+Item = Query()
```

Comment:
Unify `Item = Query()` and `Product = Query()` into a single Query() object, and replace all previous instance of `Item` and `Product`. 
E.g. :
<pre>
QUERY = Query()

def get_order(id):
    return order_db.get(Item.order_id == id)
</pre>